### DISCUSSION TITLE
How many Matlab compiler SDK are necessary to handle multiple device with EDGEX

### DISCUSSION THREAD

**Discussion by damianorossi90** [edgexfoundry/community#307] [Q&A]
Hi,
I'm gonna describe you my scenario:

- I have several matlab script
- I can dockerize them (via Matlab compiler SDK)
- I can deploy the dockers to our N edge computers associated to each one of our N plants
![Capture11](https://github.com/user-attachments/assets/bdb673da-f175-4601-8811-644805fb2703)

Questions:

1. Can the dockerized matlab scripts on the edges, run at the same time on all N devices **using just a single license** with SDK compiler?
2. Do we need N licenses with SDK Compiler to keep in RUN the computational routines on the N edge computers at the same time?

Thank you in advance for support.
Kind regards

**Comment by jiekechoo** [comment]
I don't understand you. Please decribe the `license`.

**Comment by cloudxxx8** [comment]
I am not sure whehter your questions are related to EdgeX
